etc/devices/redfishpower-cray-ex-rabbit.dev

```diff
@@ -0,0 +1,160 @@
+# Support for Redfish Rest Interface
+#
+# How blades and switches in a HPE Cray Supercomputing EX Chassis
+# (abbreviated CrayEX) are populated has a large effect on
+# configuration.  Tweaks to this device file may be needed.  See the
+# section "UPDATING REDFISHPOWER DEVICE FILES" in redfishpower(8) for
+# additional tips.
+#
+# - Set your system's username/password in the login section of
+#   each specification below.
+#
+# - Assuming all blades are populated with nodes, switch slots 0-3 are
+#   populated with switches, and switch slots 4 & 7 are populated by a
+#   rabbit, configure in Powerman like below.
+#
+#   N.B. A rabbit takes up 4 switch slots but only utilizes power from
+#   slots 4 & 7.  Slot 4 powers storage/drives, slot 7 powers processor.
+#
+#   Take special notice to the order of hosts listed with `-h` as the
+#   order matters (plugnames are mapped to hosts via indices).  The cmm
+#   should be listed first with the 16 nodes second.
+#
+#   include "/etc/powerman/redfishpower-cray-ex-rabbit.dev"
+#   device "chassis0" "cray-ex" "/usr/sbin/redfishpower -h cmm0,pmynode[0-15],rabbit |&"
+#   node "cmm0,myperif[0-3],rabbit-storage,rabbit-blade,myblade[0-7],mynode[0-15],rabbit" "chassis0" "Enclosure,Perif[0-4,7],Blade[0-7],Node[0-16]"
+#
+# - If your chassis is not fully populated, put placeholder hosts in
+#   the redfishpower hosts configuration.  Adjust plugs when
+#   configuring specific targets.  For example, let's say blades[4-7]
+#   are not populated (thus nodes[8-15] also do not exist).
+#
+#   device "chassis0" "cray-ex" "/usr/sbin/redfishpower -h cmm0,pnode[0-7],unused[0-7],rabbit |&"
+#   node "cmm0,myperif[0-3],rabbit-storage,rabbit-blade,myblade[0-3],mynode[0-7],rabbit" "chassis0" "Enclosure,Perif[0-4.7],Blade[0-3],Node[0-7,16]"
+#
+
+specification "cray-ex-rabbit" {
+	timeout 	100
+
+	plug name {
+		"Enclosure"
+		"Perif0"
+		"Perif1"
+		"Perif2"
+		"Perif3"
+		"Perif4"
```

Comment:
Insert comment like
```
# Perif[5-6] are rabbit storage with no independent power control
```

Reply:
Oops forgot to save an edit.  Another idea would be to just put them in their normal place, commented out e.g.
```
# Perif5 storage hardware - no power control
# Perif6 storage hardware - no power control
```